core/input/gesturehandler.js

```diff
@@ -186,14 +186,14 @@ export default class GestureHandler {
         }
 
         if (!this._hasDetectedGesture()) {
-            // Ignore moves smaller than the minimum threshold
-            if (Math.hypot(deltaX, deltaY) < GH_MOVE_THRESHOLD) {
-                return;
-            }
 
```

Comment:
This breaks two finger gestures if the first finger moves slightly before the second finger makes its initial touch. Which I'd suspect is very likely, given how sensitive touch devices are.